pkg/node/node.go

```diff
@@ -212,20 +212,23 @@ func NewBee(addr string, publicKey *ecdsa.PublicKey, signer crypto.Signer, netwo
 		cashoutService     chequebook.CashoutService
 		pollingInterval    = time.Duration(o.BlockTime) * time.Second
 	)
-	swapBackend, overlayEthAddress, chainID, transactionMonitor, transactionService, err = InitChain(
-		p2pCtx,
-		logger,
-		stateStore,
-		o.SwapEndpoint,
-		signer,
-		pollingInterval,
-	)
-	if err != nil {
-		return nil, fmt.Errorf("init chain: %w", err)
+
+	if o.SwapEnable {
```

Comment:
I think we decided to create a separate file for initializing the dev mode bee.

We should create a separate `NewBee` function itself instead of having `if-else` checks in this file. This will easily turn into the same ugliness as the standalone mode.

Reply:
Correct :+1: